source/features/rgh-linkify-yolo-issues.tsx

```diff
@@ -0,0 +1,27 @@
+import * as pageDetect from 'github-url-detection';
+
+import React from 'dom-chef';
+
+import select from 'select-dom';
+
+import features from '../feature-manager';
+
+import {wrap} from '../helpers/dom-utils';
+
+import {getRghIssueUrl} from '../helpers/rgh-issue-link';
+
+function init(): void {
+	for (const issueCell of select.all('td:nth-child(3)')) {
+		wrap(issueCell.firstChild!, <a href={getRghIssueUrl(issueCell.textContent!)}/>);
+	}
+}
+
+void features.add(import.meta.url, {
```

Comment:
This object is missing the asLongAs array with the "is Refined GitHub repo" filter

Reply:
I am working to add this `asLongAs ` array.

Reply:
I have added aslongAs array with `isAnyRefinedGitHubRepo` filter

Reply:
Almost! The specific filter should point only to this repo. Does it exist? If not, we should create it. 

"Is any Refined GitHub repo" will run also on https://github.com/refined-github/refined-github but this feature doesn't apply there